datadog_lambda/span_pointers.py

```diff
@@ -2,7 +2,10 @@
 import logging
 from typing import List
 
-from ddtrace._trace.utils_botocore.span_pointers import (
+from ddtrace._trace.utils_botocore.span_pointers.dynamodb import (
+    _aws_dynamodb_item_span_pointer_description,
+)
+from ddtrace._trace.utils_botocore.span_pointers.s3 import (
```

Comment:
Can we avoid importing at the top? And instead, do it conditionally where we need it?

Reply:
you mean move this function import down to where we're about to call the function in `_calculate_s3_span_pointers_for_object_created_s3_information`? we could, but that's a pretty funky thing to do in python. what would be the benefit? it may harm readability.